test/k8s-integration/driver.go

```diff
@@ -99,14 +118,18 @@ func pushImage(pkgDir, stagingImage, stagingVersion, platform string) error {
 		return err
 	}
 	var cmd *exec.Cmd
-	if platform != "windows" {
-		cmd = exec.Command("make", "-C", pkgDir, "push-container",
-			fmt.Sprintf("GCE_PD_CSI_STAGING_VERSION=%s", stagingVersion),
-			fmt.Sprintf("GCE_PD_CSI_STAGING_IMAGE=%s", stagingImage))
-	} else {
-		cmd = exec.Command("make", "-C", pkgDir, "build-and-push-windows-container-1909",
+
+	cmd = exec.Command("make", "-C", pkgDir, "push-container",
+		fmt.Sprintf("GCE_PD_CSI_STAGING_VERSION=%s", stagingVersion),
+		fmt.Sprintf("GCE_PD_CSI_STAGING_IMAGE=%s", stagingImage))
+	err = runCommand("Pushing GCP Container", cmd)
+	if err != nil {
+		return fmt.Errorf("failed to run make command: err: %v", err)
+	}
+	if platform == "windows" {
+		cmd = exec.Command("make", "-C", pkgDir, "build-and-push-windows-container-ltsc2019",
```

Comment:
Can you explain why we need to do push-container first?

Reply:
this command build and push container image together for windows, I update the runCommand message to make it clear.

Reply:
Ok I understand now. We still need to build and push LInux images because the controller part still needs to run on a Linux machine.

Reply:
yes, linux image build and push will be always triggered.